packages/components/src/molecules/Panels/args.tsx

```diff
@@ -24,6 +24,20 @@ export const AllRowsArgs: ResizableRowsPanelType = {
   },
 };
 
+export const ClosableLeftColumnArgs: ResizableColumnsPanelType = {
+  left: <div style={{ background: "#002B7F", height: "100%", color: "#fff", padding: 15 }}>Left Panel</div>,
+  center: <div style={{ background: "#FCD116", height: "100%", padding: 15 }}>Center Panel</div>,
+  right: <div style={{ background: "#CE1126", height: "100%", padding: 15 }}>Right Panel</div>,
+  height: 500,
```

Comment:
Better to indicate default size term like "500px" or "500rem" here it is obvious that its px but for small number it might be confusing.

Reply:
Yes, but since this are just args for the storybook it shouldn't matter that much...